src/server/standard_address_space_part8.cpp

```diff
@@ -699,7 +699,7 @@ namespace OpcUa
         attrs.Type = ObjectID::String;
         attrs.Rank = 1;
         attrs.IsAbstract = false;
-        attrs.Dimensions = std::vector<uint32_t>({0});
+		attrs.Dimensions = { 0 };
```

Comment:
> Understood. But being managed by git, address_space_partXXX files should
also be changed after generate_address_space.py being changed. Is it right?
Or remove them from the repository and automatically generate them using
CMake?

We discussed this last year and we decided to make git manage both the
generator his (,py) and the generated code (h and cpp). Then it is easier
to see changes. But all generated files should start with a warning that
they are auto generated. Is it missing?

>
> About C2440. I don't think attires.Dimensions = { 0 }; is an implicit
casting. It should use vector::operator=(std::initializer_list<T> ilist) to
reassign attires.Dimensions. So, my version is concise and correct.

If you can make the change in the python file and it does not break build I
am fine.